crates/holochain/src/conductor/conductor.rs

```diff
@@ -2253,6 +2253,49 @@ mod misc_impls {
             Ok(serde_json::to_string_pretty(&out)?)
         }
 
+        /// Create a JSON dump of the conductor's state
+        pub async fn dump_conductor_state(&self) -> ConductorApiResult<String> {
```

Comment:
This looks good but is quite a bit of inline code. I agree this is the best way to avoid polluting the global scope in this file. But the more stuff like this we add the more I'd like to break this file down into smaller pieces